exercises/elm-1-b/src/Pizza.elm

```diff
@@ -0,0 +1,13 @@
+module Pizza exposing (Pizza, retrieveMenu)
+
+
+type alias Pizza =
+    { name : String
+    , vegetarian : Bool
+    , price : Int
+    }
+
+
+retrieveMenu : String -> List Pizza
+retrieveMenu stringMenu =
+    Debug.todo "retrieveMenu"
```

Comment:
Hi Matthieu, this is meant to return "Please implement the `retrieveMenu` function". It is good if this makes it in to the output somehow, although it looks like this will be difficult in this case.

Cheers, Cedd

Reply:
Ok, I've updated the `Debug.todo` message to say exactly that. If I've understood correctly (@ErikSchierboom?), exercism will display the error in the message returned to the student. In this case the error becomes:
> This test failed because it threw an exception: "Error: TODO in module `Pizza` on line 13
> Please implement the `retrieveMenu` function"

Reply:
@mpizenenberg Well, if the test runner correctly captures the error and returns it as part of the `results.json` document, then yes.

Reply:
Charles care is integrating the test runner. I emailed him last week to see if there was anything we could do to help, but I don't think there is (not yet anyway). He was waiting for a website update in order to do something. It seems to work locally for me, but obviously I can't test it within the exercism system.